applications/dashboard/controllers/class.logcontroller.php

```diff
@@ -241,6 +244,47 @@ public function record($recordType, $recordID, $page = '') {
         $this->render();
     }
 
+    /**
+     * Access the log history of a specific discussion thread
+     *
+     * @param string $recordID
+     * @param string $page
+     */
+    public function discussion(string $recordID, string $page = '') {
```

Comment:
This shares an awful lot in common with `LogController::record`. Is there any way to reduce the code duplication being introduced?